src/GoldMiner/Gold.java

```diff
@@ -0,0 +1,129 @@
+package GoldMiner;
+
+import java.awt.Color;
+import java.util.ArrayList;
+import java.util.List;
+
+import edu.macalester.graphics.CanvasWindow;
+import edu.macalester.graphics.Ellipse;
+import edu.macalester.graphics.GraphicsObject;
+import edu.macalester.graphics.Rectangle;
+
+/**
+ * This file create a group of Gold pieces with different size and position. Three Classes or one?
+ */
+
+public class Gold {
+
+    private static final Color GOLDEN = new Color(255, 215, 0);
+    private static final Color DIAMOND = new Color(185, 242, 255);
+
+    private double x; // x-coordinate of each mineral
+    private double y; // y-coordinate of each mineral
+
+    public Rectangle diamond;
+    public Rectangle diamond1;
+    public Rectangle diamond2;
+
+    public List<GraphicsObject> threekindsOfMineral = new ArrayList<>();
+    // This list contains three kinds of minerals of different size.
+
+    public List<GraphicsObject> finalMineral = new ArrayList<>();
+    // This list contains the minerals that are chosen randomly and added to the canvas window.
```

Comment:
Instance variables should be private. The fact that you’ve had to make them public is a sign that you haven’t quite worked out a good division of labor among your classes.